<launch>
    <arg name="target_frame"               default="/map" />
    <arg name="source_frame"               default="/base_footprint" />
    <arg name="use_odom_topic"             default="/robot_position_in_map" />
    <arg name="amclodom_frequence"         default="0.1" />
    <node pkg="amcl_odom" type="amcl_odom.py" name="amcl_odom" output="screen" respawn="true">
        <param name = "target_frame"           value = "$(arg target_frame)" />
        <param name = "source_frame"           value = "$(arg source_frame)" />
        <param name = "use_odom_topic"         value = "$(arg use_odom_topic)" />
        <param name = "amclodom_frequence"     value = "$(arg amclodom_frequence)" />
    </node>
</launch>
